### PR TITLE
Add bi2a-iidx support for aciotest

### PR DESCRIPTION
**Pull request by MatryoshkaLJY**

### CODE REVIEW

File: src/main/aciotest/main.c [hunk 1]
```diff
@@ -53,11 +55,15 @@ static bool aciotest_assign_handler(
     }
 
     if (product_type == AC_IO_NODE_TYPE_BI2A) {
+        printf("Please assign bi2a mode, 0 for SDVX and 1 for IIDX, then press ENTER:");
```
Comment (icex2) [inline src/main/aciotest/main.c:58]: I guess that works, but compared to the other ways it detects different devices, it feels a bit off.
Do we ever have multiple acio devices on the same bus with any BI2A device? To my knowledge, the BI2A device is always on its own bus exclusively (please correct me here if I am wrong).
Assuming that's the case, I suggest to extend the tool rather to allow selecting different "kinds" of devices.

Examples how to call `aciotest`:
* `aciotest.exe COM1 57600` for a "generic" acio device that does the entire "go by product code" thing
* `aciotest.exe COM1 57600 bi2a-iidx` for the iidx BI2A mode
* `aciotest.exe COM1 57600 bi2a-sdvx` for the sdvx BI2A mode

Mind tweaking the tool a little to have it work like that?

Comment (MatryoshkaLJY) [reply src/main/aciotest/main.c]: Sorted!

File: src/main/aciotest/main.c [hunk 2]
```diff
@@ -16,7 +18,7 @@
 #include "util/log.h"
 
 static uint8_t aciotest_cnt = 0;
-static uint8_t bi2a_mode = 0;
+static uint8_t bi2a_mode = 255;
```
Comment (icex2) [inline src/main/aciotest/main.c:21]: Non-blocking remark to take away as a learning: An enum would improve readability here, e.g.

```
enum BI2A_MODE {
    BI2A_MODE_INVALID = 0,
    BI2A_MODE_SDVX = 1,
    BI2A_MODE_IIDX = 2,
};
```

If you want, you can follow up with another small refactoring PR to improve this. The current solution is fine though considering it's just two modes now and scoped to a single file.

**Review by icex2 — APPROVED** [review]
Looking good, thanks for following up on the suggestion.

**Comment by icex2** [comment]
"This branch cannot be rebased due to conflicts"

Not sure where the conflicts are. It doesn't mention any specific files. Can you check on your end if you are up to date with [djhackersdev:master](https://github.com/djhackersdev/bemanitools/tree/master) and fix that if needed?

Otherwise, I can't merge it.

File: src/main/aciotest/main.c (hunk 2 above)
Comment (MatryoshkaLJY) [reply src/main/aciotest/main.c:21]: Sure, I'll do it later

**Comment by icex2** [comment]
Thanks for following up. Merged.

**Pull request merged**